home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/NewsApiIT.java

```diff
@@ -0,0 +1,110 @@
+package com.softserveinc.ita.homeproject.api.tests;
+
+import java.util.HashMap;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import com.softserveinc.ita.homeproject.ApiClient;
+import com.softserveinc.ita.homeproject.ApiException;
+import com.softserveinc.ita.homeproject.ServerConfiguration;
+import com.softserveinc.ita.homeproject.api.NewsApi;
+import com.softserveinc.ita.homeproject.model.CreateNews;
+import com.softserveinc.ita.homeproject.model.ReadNews;
+import com.softserveinc.ita.homeproject.model.UpdateNews;
+
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class NewsApiIT {
+
+    private final NewsApi newsApi;
+
+    {
```

Comment:
What is it about moving this code block to util class?

Reply:
Done